From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Sun, 20 Aug 2023 03:38:12 +0200
Subject: po: Update POTFILES.in

We include a .desktop file for the Extensions D-Bus
service, to provide name and icon to the preference
dialog when used without a parent window.

However we missed adding it to POTFILES.in, so it is
currently untranslated, whoops.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2904>
(cherry picked from commit 3f0d3381eeb7faf295e9159a7ea86fec29bd9806)
---
 po/POTFILES.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/po/POTFILES.in b/po/POTFILES.in
index 93ad531..7bc5ecf 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -5,6 +5,7 @@ data/50-gnome-shell-screenshots.xml
 data/50-gnome-shell-system.xml
 data/org.gnome.Shell.desktop.in.in
 data/org.gnome.shell.gschema.xml.in
+data/org.gnome.Shell.Extensions.desktop.in.in
 data/org.gnome.Shell.PortalHelper.desktop.in.in
 js/dbusServices/extensions/ui/extension-error-page.ui
 js/gdm/authPrompt.js
